### PR TITLE
README translation to english and API request update.

### PR DESCRIPTION
**Pull request by gfpoliva**

### CODE REVIEW

**Review by GMBruno — APPROVED** [review]

File: README-EN.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+You can base your UI in this mockup, but feel free to develop whatever you want!
+![mockup](https://raw.githubusercontent.com/myfreecomm/desafio-mobile-ios/master/mockup-ios.png)
+
+## Project requirements:
+* Target: > iOS 9.*
+* Use .gitignore
+* Storyboard and autolayout
+* Cocoapods or Carthage for dependencies
+* Alamofire or an network framework
```
Comment (bernardoroll) [inline README-EN.md]: Alamofire or A network framework, not AN

File: README-EN.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* You use RxSwift
+* Use CoreData / Realm / Firebase to locally persist
+* Your app be universal
+* Image cache. Ex: SDWebImage/Kingfisher/AlamofireImage
+* Dependency Injection
+
+## Test
+Your project will be evaluated according to the following criteria:
+* Does your project have the basics requirements?
+* Do you documented your project (configuration, environment, code)?
```
Comment (bernardoroll) [inline README-EN.md]: Did you document

File: README-EN.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# iOS Challenge
+
+The main goal of this challenge is allow us to know better your skills.
+This challenge must be done completely by yourself.
```
Comment (bernardoroll) [inline README-EN.md]: ...must be completely done by yourself.

**Review by bernardoroll — CHANGES_REQUESTED** [review]

File: README-EN.md (hunk 3 above)
Comment (gfpoliva) [reply README-EN.md]: Fixed in ca529bdf5eb31aec656dd6e41df110c39955f2aa.

File: README-EN.md (hunk 1 above)
Comment (gfpoliva) [reply README-EN.md]: Fixed in ca529bdf5eb31aec656dd6e41df110c39955f2aa.

File: README-EN.md (hunk 2 above)
Comment (gfpoliva) [reply README-EN.md]: Fixed in ca529bdf5eb31aec656dd6e41df110c39955f2aa.

**Pull request merged**